src/emulate.c

```diff
@@ -38,13 +38,14 @@ extern struct target_ops gdbstub_ops;
 #define IF_imm(i, v) (i->imm == v)
 
 /* RISC-V exception code list */
-#define RV_EXCEPTION_LIST                                       \
-    _(insn_misaligned, 0)  /* Instruction address misaligned */ \
-    _(illegal_insn, 2)     /* Illegal instruction */            \
-    _(breakpoint, 3)       /* Breakpoint */                     \
-    _(load_misaligned, 4)  /* Load address misaligned */        \
-    _(store_misaligned, 6) /* Store/AMO address misaligned */   \
-    _(ecall_M, 11)         /* Environment call from M-mode */
+#define RV_EXCEPTION_LIST                                           \
+    IIF(RV32_HAS(EXT_C))                                            \
```

Comment:
Use `/* clang-format off */` and `/* clang-format on */` to enforce consistent style.